lib/ssh/doc/guides/using_ssh.md

```diff
@@ -17,26 +17,24 @@ limitations under the License.
 
 %CopyrightEnd%
 -->
-# Getting Started
-
-## General Information
+# Examples
 
 The following examples use the utility function `ssh:start/0` to start all
 needed applications (`crypto`, `public_key`, and `ssh`). All examples are run in
-an Erlang shell, or in a bash shell, using _openssh_ to illustrate how the `ssh`
+an Erlang shell, or in a bash shell, using **OpenSSH** to illustrate how the `ssh`
 application can be used. The examples are run as the user `otptest` on a local
-network where the user is authorized to log in over `ssh` to the host _tarlop_.
+network where the user is authorized to log in over `ssh` to the host **tarlop**.
```

Comment:
I guess it used to be hostname for some writing examples.
any suggestion on better example hostname?

Reply:
`my_ssh_host` or sth of the like maybe?

Reply:
What a good example hostname is may also depend on whether you're in a client or daemon example.

Reply:
I think Tarlop might be a name of some daemon (do not really know from where)  I just remembered from the days when I played WOW and played a warlock that had daemon companions that one of them was called Tarlop ;).  But maybe it would be better to use some fake DNS name such as  ssh.example.com?